src/MainWindow.cc

```diff
@@ -459,6 +459,18 @@ MainWindow::MainWindow(const QStringList &filenames)
 	connect(this->helpActionCheatSheet, SIGNAL(triggered()), this, SLOT(helpCheatSheet()));
 	connect(this->helpActionLibraryInfo, SIGNAL(triggered()), this, SLOT(helpLibrary()));
 	connect(this->helpActionFontInfo, SIGNAL(triggered()), this, SLOT(helpFontInfo()));
+		
+	// Checks if the Documentation has been downloaded and hides the Action otherwise
+	fs::path resPath = PlatformUtils::resourcePath("resources");
+    fs::path fullPath = resPath/"docs"/"OpenSCADUserDocs"/"openscad_docs"/"OpenSCAD_User_Manual.html";
+	if (fs::exists(fullPath) && fs::is_regular_file(fullPath)) 
```

Comment:
This looks fine on it's own but as we have an almost identical part in UIUtils, there's the opportunity to move everything to this class. So it could have an internal function generating the path and provide 2 methods to the outside.
- hasOfflineUserManual() - for use here in the `if`
- openOfflineUserManualURL() - this will remain as it is now